FrontEnd/src/pages/AboutUsPage/AboutUs.module.css

```diff
@@ -0,0 +1,126 @@
+
+.about-us-section {
+    min-width: 100vw;
+    background: var(--about-us-background-color);
+    justify-content: center; 
+    align-items: center;
+}
+
+.about-us-section-main {
+    box-sizing: border-box;
+    padding: 40px 0;  
+    align-items: center;
+    display: flex;
+    flex-direction: column;
+    gap: 24px;
+}
+
+.about-us-section-content__header {
+    color: var(--about-us-header-color);
+    margin-bottom: 24px;
+    margin-top: 40px;
+    font-family: var(--font-main);
+    font-size: 24px;
+    font-weight: 700;
+    line-height: 1.2;
+    letter-spacing: 0.24px;
+    text-align: center;
+    width: 100%;
+}
+
+.about-us-section-content {
+    box-sizing: border-box;
+    min-width: 345px;
+    width: 375px;
+    padding: 0 15px;
+    display: flex;
+    justify-content: space-between;
```

Comment:
There is a redundant justify-content property. It has no effect on mobile and tablet screens. And it is only applied for screens > 1200px in media query.